docs/DeveloperGuide.md

```diff
@@ -553,57 +486,3 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * **GUI**: Graphical user interface
 * **Relative File Path**: A path that is relative to a current directory, it is combined with another file path to access a file.
 * **Absolute File Path**: The complete details required to locate the file or folder, starting with the root element.
-
---------------------------------------------------------------------------------------------------------------------
-
-## **Appendix: Instructions for manual testing**
```

Comment:
Are we deleting appendix for manual testing altogether or will we add it later?

Reply:
I don't think we will be doing manual testing anymore. Moreover, this appendix is for us to read and probably not suitable to be included in DG.